### PR TITLE
cutting corpsman prep. and other medical prep overhaul.

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
replace the Peri+ qc+ synaps
injector in belts with a syringe case holding said items.

replace the hypo corps pouch

with a medkit pouch holding
the same hypo the hypo pouch had
and a syringe case with ( QC+ Peri+ Synaps)

essentially just a neater configuration of stuff using already avaibale IC stuff

lowers the price of big hypo so its inline with its value

removes the scamm pouch from corpsman vendor

essential kit now comes with a medkit pouch holding the same stuff a advanced kit would hold. (so its the same thing. but.. you can place in the pouch.. if you want instead of needing to move it somewhere cause the medkit doesnt fit anywhere by itself)

lets the medkit pouch hold the same stuff the advanced medkit can (except for ammo)

lets the corpsman white webbing hold tweezers

adds imialk pillbottles (and adds them to the medic lifesaver)

update combat hypos so they have tramadol ((like the advanced combat injectors))

adds iron pill bottles (and adds it to medic vendor)

add the following items in the medic vendor
combat injector pouches  = 4,  (with 3 combat injectors inside each)
bicaridine pill_bottle = 4,
kelotane pill_bottle = 4,
tramadol pill_bottle = 4,
tricordrazine pill_bottle = 4,
alkysine = 4,
imidazoline = 4,
iron pill_bottle = 4
brute kits = 16,
burn kits = 16,
splints = 8,

adds the ability to restock ( peri+ qc+ dex+ and synaps injectors) in the medic vendor

doubles the quantity of pouches in medic vendor

TLDR:   creates a imialk and a iron bottle of pills.   using the already existing and avaible pouches i optimized some of the aspects of the medical loadout. adding some of the more. specific but yet common medical items for the medical vendor (the one that is exclusive to corps) to help corps prep in their bunkers,    (the belt changes only aftects corpsman the rest of lifesaver belts stays the same), fixes the tweezers storage. lowerst the price of big hypo to 5. and removes the scammy medkitpouch from prep.

## Why It's Good For The Game
QoL for corpsman and helps cut prep time

## Changelog
:cl:
add: pouch holding Peri+ qc+ synaps injectors
add: medkitpouch with the contents of the old corps hypo pouch
add: imialk and iron pillbottles
add: imialk bottle to the medic lifesaver belt
add: iron, alk, imi, bica, kelo, tram, trico, pillbottles to medic vendor
add: brute kits, burn kits, splints to the medic vendor
add: adds bottles ( bica, kelo, tram, inap, dylo) and beakers (60u) (120u if hacked) to medic vendor for small mixes in prep
add: adds the ability to restock ( peri+ qc+ dex+ and synaps injectors) in the medic vendor
del: Peri+ qc+ synaps injectors from the lifesaver belt
del: combat injectors from the medic lifesaver belt
del: hypo corps pouch
tweak: essential corpsman kit now comes with a medkitpouch with the same contents of the now removed adv kit
tweak: peri+ qc+ and synps are now inside a pouch in the medic storage rig
tweak: doubles the quantity of pouches in medic vendor
balance: lowers the price of big hypo from 10 to 5
balance: lets the medkit pouch hold the same stuff the advanced medkit can (except for ammo)
balance: lets the corpsman white webbing hold tweezers
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -39. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by fernandos13** [comment]
submit your loadout to see if i can catter to your personal stuff (i'm not gonna to code EVERYONE loadout in the machine. BUT if every corpsman agrees we need to idk ditch the hypervene injector for a pillbottle. or even ditch the splinter for a cryobag. we can discuss this sorta stuff)

also post custom mixes.. SURE i might not code in mega master UNGA JUICE. with 10u of each chem in the game
but if 100% of corpsman go up to chemstry and make a bottle of imialk. i might code it in

so .. post here feedback cause i'm in dire need of it

**Comment by Swagile** [comment]
Wish I could give feedback but I don't play corpsmen enough to give you anything of substance.

Ask Aleksy on discord, he plays the shit out of corpsmen, maybe he has some input.

**Comment by Losenis** [comment]
Lifesaver belt has a shitton of combat injectors that I frankly find useless, since if you needed use out of them you could just have one single slot of usage by getting a hypospray and loading it up with the assorted chems. This, however, is redundant since a lot of corpsmen just make their own chem cocktail injectors anyways, so they're rather useless unless all other options run out, but a single use item taking up an entire slot is already tremendously inefficient given you can just make pills of chems you don't need dispenser energy for due to the vendor.

Because of this, I'd highly suggest replacing the three combat injectors in the lifesaver belt with a pill bottle of alkysine, a pill bottle of imiwhateveritwascalled, and a bottle of dexaline. I don't believe the pill bottles for iron exist ingame, and I don't know how difficult it'd be to code them, but if effort allowed I'd also suggest removing one combat injector syringe case for an iron pill bottle of 15u iron each. That way corpsmen are able to treat brain/eye damage with chems that already exist in the marinemed, so it wouldn't be a big balance shift to include them in, as well as actually halting/delaying more oxyloss and bloodloss. Given peri+ is refillable from vendor now, as well as qc+ on top of having more than enough from pouches, they aren't really that needed other than the injectors coming with it.

Those are some things that take up a lot of my prep time. Going into chem and, assuming nobody is there, making:
Pill bottle of Iron (Wave goodbye to dispenser energy for 240u of it if there's a lot of corpsmen wanting the same)
Pill bottle of Peri+ (takes more time than you'd think due to fiddling with grinder and GOING ALL THE WAY TO UPPER STORAGE IN SULACO DUE TO NOT ENOUGH PHORON SHEETS IN CHEM, juggling low quantity peridaxon bottles from vendor, and getting ryetalin done, then adding dylovene to the mix to counter the toxin metabolization)
Pill bottle of imialky (Takes up chem dispenser energy by recipe since I'm not spending even more time than needed)
Pill bottle of super unga mix of like 10 different cocktails, this obviously will still happen due to being a custom personal luxury endeavour rather than a critical corpsman necessity
Tricord pill bottles assuming they aren't all taken from vendor/med storage, since I use tricord in a hypospray for normal marine healing

And all of this after gearing up in prep for a normal loadout and packing ammo, assuming I run a normal memorized loadout rather than stop and think "what if i wanted to try something else this round". Joining by readying up at roundstart still has me miss out the first alamo drop and shutters opening sometimes.

**Comment by BraveMole** [comment]
- Lifesaver belt has a shitton of combat injectors that I frankly find useless
Same, utterly useless

- Because of this, I'd highly suggest replacing the three combat injectors in the lifesaver belt with a pill bottle of alkysine, a pill bottle of imiwhateveritwascalled, and a bottle of dexaline
Strongly agrees

The rest of what Losenis said makes sense, even though it feels a bit more powercreep.

I'd say changing life saver belt to something noob friendly is also important, because in your first game you will have no idea on what meds to take. If the default option is already a good one, this will make new medics less usesless

**Comment by Losenis** [comment]
I took into consideration things like powercreep so I limited proposed changes to what already is available without needing to specifically make it from scratch, like bottles of alkysine and imi (separate, not actual imialky bottle), hypervene which I forgot about, dexaline which is weird to not have it given it's the fourth damage type, and ideally iron since it's a base reagent rather than something requiring a recipe. The core idea is making the corpsman be equipped by default to take on 99% of the situations he'll find himself in, even if not optimally, since minmaxing the loadout will fall under trading more prep time for more control.

For example, imi and alky pill bottles would be separate which already comes in the marinemed (none of us knew to take them when we started out as corpsmen, let's be real here), with the corpsman having the choice to spend time mixing them to save on a slot. Peri+ injector can be refilled after use, but the corpsman also has the choice to forego it and make a pill bottle of 1u peri+ 14u dylov if he wanted to. For how powerful a loadout might be in the right hands, there aren't that many people out there to use it to its full potential, and even then they'd minmax the loadout in chem anyway. Default loadout should be a "It works, but customizing it can make it better" thing rather than "I outright am not equipped to deal with some situations". The default peri+ can only heal so much brain damage before it runs out and the actual pill bottle of alkysine has to come in, which corpsmen don't start with.

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -260,7 +260,15 @@
 	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
+/obj/item/storage/syringe_case/corps
```
Comment (TiviPlus) [inline code/game/objects/items/storage/firstaid.dm:263]:
```suggestion
	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)

/obj/item/storage/syringe_case/corps
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]
A) give me a tldr that isnt 4 pages
B) Follow the PR format guidelines

**Comment by Chaznoodles** [comment]
syringes/syringe cases ungood

**Comment by fernandos13** [comment]
> A) give me a tldr that isnt 4 pages
> B) Follow the PR format guidelines


will do when finishing up the draft. 
 (that and the changelog. first i wanna know and worry about what people are and are not getting so i can actually WRITE it down )
 
 otherwise it will be a always changing thing

**Comment by BraveMole** [comment]
> will do when finishing up the draft.

If it's draft set it as draft, it's a setting

**Comment by fernandos13** [comment]
> > will do when finishing up the draft.
> 
> If it's draft set it as draft, it's a setting

i did it on the title how do i do it on the.. setting?

**Comment by BraveMole** [comment]
image
 Here

**Comment by fernandos13** [comment]
OH thanks! now its draft

**Comment by fernandos13** [comment]
new changelog

**Comment by fernandos13** [comment]
> A) give me a tldr that isnt 4 pages

creates a imialk and a iron bottle of pills.   using the already existing and avaible pouches i optimized some of the aspects of the medical loadout. adding some of the more. specific but yet common medical items for the medical vendor (the one that is exclusive to corps) to help corps prep in their bunkers,    (the belt changes only aftects corpsman the rest of lifesaver belts stays the same), fixes the tweezers storage. lowerst the price of big hypo to 5. and removes the scammy medkitpouch from prep.

**Comment by fernandos13** [comment]
well no more feedback guess we ready to launch out of draft

**Comment by MemesLauncher** [comment]
Shut it and take my money, all my yes!
Although one thing I'd add (as said in the comments before me) is to replace the useless injectors of the lifesaver with spaceacilin, dex, hypervene, imidazoline (eye fix) and alkysine (brain fix) as those are pretty useful, and your heart breaks when your local new medic doesn't have any.

Another thing could be adding more pouches in the med prep vendors (the surplus one in spring for exemple, 4 isn't enough which warrants a trip to the surplus clothing vendor).

**Comment by fernandos13** [comment]
> Shut it and take my money, all my yes!
> Although one thing I'd add (as said in the comments before me) is to replace the useless injectors of the lifesaver with spaceacilin, dex, hypervene, imidazoline (eye fix) and alkysine (brain fix) as those are pretty useful, and your heart breaks when your local new medic doesn't have any.
> 
> Another thing could be adding more pouches in the med prep vendors (the surplus one in spring for exemple, 4 isn't enough which warrants a trip to the surplus clothing vendor).

```
-	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
-	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
-	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
+	new /obj/item/storage/pill_bottle/hypervene(src)
+	new /obj/item/storage/pill_bottle/dexalin(src)
+	new /obj/item/storage/pill_bottle/imialk
```
	
	
	that belt change is in the code. albeit with hypervene instead of spaceacillin
	
	about the pouches good point!

File: code/game/objects/items/reagent_containers/pill.dm [hunk 2]
```diff
@@ -241,3 +241,14 @@
 	//pill_desc = "An ultrazine pill. A highly-potent, long-lasting combination CNS and muscle stimulant. Extremely addictive."
 	list_reagents = list(/datum/reagent/medicine/ultrazine = 5)
 	pill_id = 21
+
+
+/obj/item/reagent_containers/pill/imialk
+	pill_desc = "An imialk pill, a old mix made to heal both eyes and brain."
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/pill.dm:247]: say what is in it

File: code/game/objects/items/reagent_containers/pill.dm [hunk 3]
```diff
@@ -241,3 +241,14 @@
...
+	list_reagents = list(/datum/reagent/medicine/imidazoline = 7.5,/datum/reagent/medicine/alkysine = 7.5)
+	pill_id = 17
+
+/obj/item/reagent_containers/pill/iron
+	pill_desc = "An raw iron pill, terragov solution for all the bloodloss, not be confused by coins."
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/pill.dm:252]:
```suggestion
	pill_desc = "An raw iron pill, terragov solution for all bloodloss, big and small, not be confused by coins."
```

File: code/game/objects/items/storage/firstaid.dm [hunk 4]
```diff
@@ -429,6 +438,18 @@
 	max_storage_space = 7
 	pill_type_to_fill = /obj/item/reagent_containers/pill/zoom
 
+/obj/item/storage/pill_bottle/imialk
+	name = "imialk pill bottle"
+	desc = "Contains pills that heal brain and eye damage."
```
Comment (TiviPlus) [inline code/game/objects/items/storage/firstaid.dm:443]: ditto see above

File: code/game/objects/items/storage/firstaid.dm [hunk 5]
```diff
@@ -429,6 +438,18 @@
 	max_storage_space = 7
 	pill_type_to_fill = /obj/item/reagent_containers/pill/zoom
 
+/obj/item/storage/pill_bottle/imialk
```
Comment (TiviPlus) [inline code/game/objects/items/storage/firstaid.dm:441]: And these paths should be imialky as that's the more common name

File: code/game/objects/items/storage/pouch.dm [hunk 6]
```diff
@@ -501,6 +506,11 @@
 	new /obj/item/stack/medical/splint(src)
 
 /obj/item/storage/pouch/medkit/equippedcorpsman/Initialize()
+	. = ..()
+	new /obj/item/reagent_containers/hypospray/advanced/combat_advanced(src)
+	new /obj/item/storage/syringe_case/corps(src)
+
+/obj/item/storage/pouch/medkit/medkit/Initialize()
 	. = ..()
```
Comment (TiviPlus) [inline code/game/objects/items/storage/pouch.dm:514]: /medkit/medkit

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 7]
```diff
@@ -583,14 +583,36 @@
...
 		/obj/item/storage/firstaid/adv = 4,
-		/obj/item/storage/pouch/medical = 4,
-		/obj/item/storage/pouch/medkit = 4,
-		/obj/item/storage/pouch/magazine/large = 4,
-		/obj/item/storage/pouch/magazine/pistol/large = 4,
+		/obj/item/storage/pouch/medical = 8,
+		/obj/item/storage/pouch/medkit = 8,
+		/obj/item/storage/pouch/magazine/large = 8,
+		/obj/item/storage/pouch/magazine/pistol/large = 8,
+		/obj/item/storage/pouch/pistol = 8,
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/marine_vending.dm:590]: No

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 8]
```diff
@@ -583,14 +583,36 @@
...
+		/obj/item/storage/pill_bottle/kelotane = 4,
+		/obj/item/storage/pill_bottle/tramadol = 4,
+		/obj/item/storage/pill_bottle/tricordrazine = 4,
+		/obj/item/storage/pill_bottle/alkysine = 4,
+		/obj/item/storage/pill_bottle/imidazoline = 4,
+		/obj/item/storage/pill_bottle/iron = 4,   //we will add more if needed later
+		/obj/item/stack/medical/advanced/bruise_pack = 16,
+		/obj/item/stack/medical/advanced/ointment = 16,
+		/obj/item/stack/medical/splint = 8,
+		/obj/item/reagent_containers/glass/beaker = 8, //so medics can make SMALL mixes in their quarters
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/marine_vending.dm:604]: No, add a beaker to prep then

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 9]
```diff
@@ -583,14 +583,36 @@
...
+		/obj/item/reagent_containers/glass/bottle/tramadol = 4,
+		/obj/item/reagent_containers/glass/bottle/inaprovaline = 4,
+		/obj/item/reagent_containers/glass/bottle/dylovene = 4,
+		/obj/item/reagent_containers/hypospray/autoinjector/dexalinplus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/quickclotplus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/synaptizine = 0,
 	)
-	contraband = list(/obj/item/reagent_containers/blood/OMinus = 1)
+	contraband = list(/obj/item/reagent_containers/blood/OMinus = 1, /obj/item/reagent_containers/glass/beaker/large = 4) //if medics REALLY want to make mixes in here. we will get in game feedback
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/marine_vending.dm:615]: ditto

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 10]
```diff
@@ -583,14 +583,36 @@
...
+		/obj/item/reagent_containers/glass/beaker = 8, //so medics can make SMALL mixes in their quarters
+		/obj/item/reagent_containers/glass/bottle/bicaridine = 4,
+		/obj/item/reagent_containers/glass/bottle/kelotane = 4,
+		/obj/item/reagent_containers/glass/bottle/tramadol = 4,
+		/obj/item/reagent_containers/glass/bottle/inaprovaline = 4,
+		/obj/item/reagent_containers/glass/bottle/dylovene = 4,
+		/obj/item/reagent_containers/hypospray/autoinjector/dexalinplus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/quickclotplus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/synaptizine = 0,
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/marine_vending.dm:613]: Why are you adding these in the first place then

File: code/game/objects/items/storage/pouch.dm [hunk 11]
```diff
@@ -488,6 +488,11 @@
 		/obj/item/flashlight/pen,
 		/obj/item/storage/pill_bottle/packet,
 		/obj/item/reagent_containers/hypospray,
+		/obj/item/storage/syringe_case,
+		/obj/item/bodybag,
+		/obj/item/defibrillator,
```
Comment (TiviPlus) [inline code/game/objects/items/storage/pouch.dm:493]: No defib

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/vending/marine_vending.dm (hunk 10 above)
Comment (fernandos13) [reply code/game/objects/machinery/vending/marine_vending.dm:612]: to allow restocking of said med

**Comment by fernandos13** [comment]
huh?

**Comment by fernandos13** [comment]
review ma please

File: code/game/objects/items/storage/belt.dm [hunk 12]
```diff
@@ -220,12 +219,13 @@
 	new /obj/item/storage/pill_bottle/inaprovaline(src)
 	new /obj/item/storage/pill_bottle/peridaxon(src)
 	new /obj/item/storage/pill_bottle/quickclot(src)
-	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
-	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
-	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
+	new /obj/item/storage/pill_bottle/hypervene(src)
+	new /obj/item/storage/pill_bottle/dexalin(src)
+	new /obj/item/storage/pill_bottle/imialky
```
Comment (TiviPlus) [inline code/game/objects/items/storage/belt.dm:224]: powercreep, no

File: code/game/objects/items/storage/pouch.dm [hunk 13]
```diff
@@ -501,6 +505,11 @@
 	new /obj/item/stack/medical/splint(src)
 
 /obj/item/storage/pouch/medkit/equippedcorpsman/Initialize()
+	. = ..()
+	new /obj/item/reagent_containers/hypospray/advanced/combat_advanced(src)
+	new /obj/item/storage/syringe_case/corps(src)
```
Comment (TiviPlus) [inline code/game/objects/items/storage/pouch.dm:510]: Powercreep, no

File: code/game/objects/machinery/vending/marine_vending.dm (hunk 10 above)
Comment (TiviPlus) [reply code/game/objects/machinery/vending/marine_vending.dm:612]: Remove it

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 14]
```diff
@@ -1318,7 +1317,7 @@ GLOBAL_LIST_INIT(available_specialist_sets, list("Scout Set", "Sniper Set", "Dem
 		/obj/item/roller,
 		/obj/item/tweezers,
 		/obj/item/reagent_containers/hypospray/advanced/oxycodone,
-		/obj/item/storage/firstaid/adv,
+		/obj/item/storage/pouch/medkit/medickit,
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:1320]: Keep this as a medkit

**Review by TiviPlus — CHANGES_REQUESTED** [review]
